README.md

```diff
@@ -42,3 +42,36 @@ Your class instance, `bme`, will now have the following methods:
 * `bme.altitude()` - return the altitude in feet, calculated against the current QNH value
 * `bme.set_qnh(value)` - set the QNH value for calculating altitude
 
+## sound
+
+For sound readings you must use the sound driver. First initialise it like so:
+
+```
+import sound
+sound = sound.sound()
+```
+
+Your class instance, `sound`, will now have the following methods:
+
+* `sound.read()` - take a reading of the sound level which you can use to compare different sound levels 
+* `sound.wait_for_double_clap()` - listen for a high sound level twice in a second
+* `sound.wait_for_clap()` - listen for a high sound level once in a second
+
+## tcs3472
+
+For tcs3472 readings you must use the tcs3472 driver. First initialise it like so:
+
+```
+import tcs3472
+
+light_sensor = tcs3472.tcs3472() 
+```
+
+Your class instance, `light_sensor`, will now have the following methods:
+
+* `light_sensor.rgb()` - returns the corrected levels of red,green and blue out of 255  
+* `light_sensor.scaled()` - return the amounts of red,green and blue on a scale of 0-1 
```

Comment:
Pedantry, but add a space after the commas

Reply:
added space